### ISSUE TITLE
[wasm] [Material] CalendarView buttons are not round

### ISSUE THREAD

**Issue by morning4coffe-dev** [unoplatform/Uno.Themes#943]
## Current behavior

Currently the CalendarView has buttons that does not have the same width/height ratio. Screenshot from Gallery:
![image](https://user-images.githubusercontent.com/78549750/222278515-417e8472-4b64-466f-b50e-4ad0ed4b32f1.png)

## Expected behavior

![image](https://user-images.githubusercontent.com/78549750/222279715-d4c050b3-fc6f-4764-b6ca-d1e9c547c570.png)

## Environment

Nuget Package:

Package Version(s):

Affected platform(s):

- [ ] iOS
- [ ] Android
- [x] WebAssembly
- [ ] UWP
- [ ] MacOS

**Comment by kazo0** [comment]
Yeah we have struggled with that for quite some time, @agneszitte-nventive had looked into it at some point but not sure what the conclusion was

**Comment by agneszitte-nventive** [comment]
> Yeah we have struggled with that for quite some time, @agneszitte-nventive had looked into it at some point but not sure what the conclusion was

@kazo0 The issue was that we also have to support stadium shapes (pill form) and not just circles for the months/years for example
![image](https://user-images.githubusercontent.com/16295702/231817372-faf8706d-c403-4b49-8f72-3de404ad0f45.png)
![image](https://user-images.githubusercontent.com/16295702/231817479-43b12735-6d46-447f-87c7-f3879a8f9d19.png)
So that need to take into account
